api/src/identity-access-management/domain/usecases/get-current-user.usecase.js

```diff
@@ -25,11 +26,17 @@ export const getCurrentUser = withTransaction(async function ({
   const user = await userRepository.get(authenticatedUserId);
   const shouldSeeDataProtectionPolicyInformationBanner = user.shouldSeeDataProtectionPolicyInformationBanner;
 
+  let anonymousUserToken = null;
```

Comment:
Cette initialisation est inutile : 
```suggestion
  let anonymousUserToken;
```